src/shared/shared-kernel/src/fh-shared-kernel.shared-kernel/OpenReferral/Repository/OpenReferralDbContextExtension.cs

```diff
@@ -0,0 +1,259 @@
+using FamilyHubs.SharedKernel.OpenReferral.Entities;
+using Microsoft.EntityFrameworkCore;
+using Attribute = FamilyHubs.SharedKernel.OpenReferral.Entities.Attribute;
+
+namespace FamilyHubs.SharedKernel.OpenReferral.Repository;
+
+public class OpenReferralDbContextExtension
+{
+    public void OnModelCreating(ModelBuilder modelBuilder)
+    {
+        // Table Mapping
+
+        modelBuilder.Entity<Accessibility>(entity =>
+        {
+            entity.ToTable("Accessibility", schema: "deds");
+            entity.HasKey(e => e.Id).IsClustered(false);
+            entity.Property(e => e.Id).ValueGeneratedNever();
+            entity.Property(e => e.Url).HasMaxLength(2048);
+        });
+
+        modelBuilder.Entity<Address>(entity =>
+        {
+            entity.ToTable("Address", schema: "deds");
```

Comment:
I think we can use `.ToSchema("deds")` instead of `ToTable("Address", schema: "deds");` to avoid having to specify the table name too :)

Failing that, this would be a good opportunity to use `nameof(Address)` rather than hard-code the string: if the Address class gets renamed, the table will also be renamed to reflect the entity, instead of the developer having to remember 🤓 

```c#
entity.ToTable(nameof(Address), schema: "deds");
```

Reply:
Doesn't seem to be a `ToSchema()` method, so went with the `nameof` way

[4234567](https://github.com/DFE-Digital/fh-services/pull/89/commits/42345678c0fe039770ef37690d471a21acc9f869)